src/main/java/com/courtcanva/ccfranchise/utils/OrderUtil.java

```diff
@@ -0,0 +1,18 @@
+package com.courtcanva.ccfranchise.utils;
+
+import com.courtcanva.ccfranchise.constants.OrderStatus;
+
+public class OrderUtil {
+
+    private OrderUtil() {
+    }
+
+    public static boolean statusValid(String test) {
```

Comment:
What is this used for?

Reply:
It's for check if the status attribute got from frontend is valid.
If you see the usage, you'll find it's used in `FranchiseeServiceTest.whenStatusParameterInvalidShouldThrowOrderStatusInvalidException()`.
the argument `test` is not good, i'll change it to `statusFromFrontend`

Reply:
becasue I need to run test

 ```
   @Test
    void whenStatusParameterInvalidShouldThrowOrderStatusInvalidException() {
        String invalidStatusValue = "invalidValue";
        assertFalse(statusValid(invalidStatusValue));
        assertThrows(OrderStatusInvalidException.class,()->franchiseeService.getOrders(1L,invalidStatusValue));
    }
```

Reply:
You don't need assetFalse here. Because when the status is invalid, the exception will be thrown.

Reply:
Simply thinking, yes, this line seams redundant  as "invalidValue" is not included in valid status enum.
I used `assertFalse `to do the check and make sure it's not included in enum.
This line of code can enhance the robustness of the test and reduce human thinking.
And in case in future "invalidValue" is used as a valid status, this line can catch the exception easily.

Reply:
1. if you use `static`, you don't need the constructor of this class
2. But I don't allow using `static` in our code
3. If this one is only used in test, it should not be put under `src/main/java`
4. But usually, if the value is from frontend, and you define it as Enum in request, Jackson will help us to check it automatically with an exception
5. less code will cause less human thinking
6. if you really want to practice. this is an example. And the truth is, we don't need this class. That's all. 
```
  public boolean statusValid(String status) {
    return Arrays.stream(OrderStatus.values()).filter(s -> s.value.equals("status")).count() != 0;
  }
```